tests/tests_ecg.py

```diff
@@ -3,7 +3,7 @@
 import neurokit2 as nk
 import matplotlib.pyplot as plt
 
-import biosppy
+# import biosppy
```

Comment:
I think this needs to be there so that it's tested against, + these packages are only downloaded for the tests on travis & co, and are not part of dependencies